src/componententity.cpp

```diff
@@ -279,8 +276,6 @@ void ComponentEntity::removeUnits(size_t index)
 {
     if (index < mPimpl->mUnits.size()) {
```

Comment:
As index is an unsigned int comparing `index >= 0` is a no-op and produces a warning from at least one compiler, that is the reason we are not comparing for positive integers.